app/browser/api/ledger.js

```diff
@@ -91,7 +90,8 @@ const clientOptions = {
   rulesTestP: process.env.LEDGER_RULES_TESTING,
   verboseP: process.env.LEDGER_VERBOSE,
   server: process.env.LEDGER_SERVER_URL,
-  createWorker: electron.app.createWorker
+  createWorker: electron.app.createWorker,
+  version: 'v2'
```

Comment:
is this what triggers the client library to use BAT?

Reply:
yes

Reply:
and we use `bat-*` libraries instead of `ledger-*`. Check in `package.json`